cmd/supernode/app/root.go

```diff
@@ -106,6 +106,11 @@ func setupFlags(cmd *cobra.Command, opt *Options) {
 
 // runSuperNode prepares configs, setups essential details and runs supernode daemon.
 func runSuperNode() error {
+	// create home dir
```

Comment:
Use `fiileutils.CreateDirectory` instead.

Reply:
Or running nginx as root can also solve this problem.